frontend/src/components/assessments/assessment-status/EditStatusButtons/UnarchiveButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import EditStatusButton from "../EditStatusButton";
+import UnarchiveModal from "../EditStatusModals/UnarchiveModal";
+
+interface UnarchiveButtonProps {
+  assessmentId: string;
+  closePopover: () => void;
+}
+
+const UnarchiveButton = ({
+  assessmentId,
+  closePopover,
+}: UnarchiveButtonProps): React.ReactElement => {
+  const [showUnarchiveModal, setshowUnarchiveModal] = React.useState(false);
+
+  return (
+    <>
+      <EditStatusButton
+        name="Unarchive"
+        onClick={() => {
+          closePopover();
+          setshowUnarchiveModal(true);
```

Comment:
nit: setShowUnarchiveModal
```suggestion
          setShowUnarchiveModal(true);
```